django/contrib/admin/templates/admin/auth/user/add_form.html

```diff
@@ -15,4 +15,4 @@
 {% block admin_change_form_document_ready %}
   {{ block.super }}
   <script src="{% static 'admin/js/unusable_password_field.js' %}" defer></script>
-{% endblock %}
+{% endblock %}
```

Comment:
```suggestion
{% endblock %}

```
(nit) revert this change